<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>328</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pbtn_start">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="toolTip">
    <string>recording</string>
   </property>
   <property name="text">
    <string notr="true">●</string>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="shortcut">
    <string>Space</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbtn_stop">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>200</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>stop</string>
   </property>
   <property name="text">
    <string>■</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcd_timer">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>200</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
   <property name="smallDecimalPoint">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pbtn_save">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>260</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Sheet Save</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>-50</x>
     <y>10</y>
     <width>491</width>
     <height>181</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="QV_plot"/>
  </widget>
  <widget class="QPushButton" name="pbtn_play">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Listen Record</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbtn_midi">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>260</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Listen MIDI</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbtn_pause">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>PAUSE</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
